src/main/webapp/style.css

```diff
@@ -116,3 +116,17 @@ h5 {
   line-height: .1;
   font-weight: bold;
 }
+*{
```

Comment:
What is this?

Reply:
The * selects all the elements to make it look the same regardless